webdriver/get_title.py

```diff
@@ -0,0 +1,282 @@
+import pytest
+import time
+
+from support.asserts import assert_error, assert_success, assert_dialog_handled
+from support.fixtures import create_dialog
+from support.inline import inline
+
+def read_global(session, name):
+    return session.execute_script("return %s;" % name)
```

Comment:
I personally don’t feel this function adds any value. `expected_title = session.execute_script("return document.title")` is not substantively longer to write than `expected_title = read_global(session, "document.title")`.

But I guess it’s fine.